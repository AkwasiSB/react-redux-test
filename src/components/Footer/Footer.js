import React from 'react';

const Footer = function() {
	return (
		<footer>
			<p>Copyrights Akwasi Sefa-Boakye. All rights reserved.</p>
		</footer>
	);
};

export default Footer;
