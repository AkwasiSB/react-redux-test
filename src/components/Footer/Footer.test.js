import Footer from './Footer';

describe('Renders Footer and it contents', () => {
    let footerWrapper;

    beforeEach(() => {
        footerWrapper = shallow(<Footer />);
    });

    it('should render Footer correctly', () => {
        expect(footerWrapper).toMatchSnapshot();
    });

    it('should display text correctly in Footer', () => {
        expect(footerWrapper.find('p').text()).toEqual('Copyrights Akwasi Sefa-Boakye. All rights reserved.');
    });
});
