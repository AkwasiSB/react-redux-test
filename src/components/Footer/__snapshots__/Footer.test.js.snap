// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Footer and it contents should render Footer correctly 1`] = `
<footer>
  <p>
    Copyrights Akwasi Sefa-Boakye. All rights reserved.
  </p>
</footer>
`;
